jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -38,8 +38,8 @@ def status(self, repo_path):
 		status = self.run(repo_path, 'status').decode('utf-8')
 		return status
 
-	def appraise_pull(self, repo_path):
-		self.run(repo_path, 'appraise', 'pull', self.remote)
+	def appraise_pull(self, git_root_dir):
+		self.run(git_root_dir, 'appraise', 'pull', self.remote)
```

Comment:
We should probably implement some sort of queue to make sure that we don't have multiple `git appraise pull` commands happening in the same repository at the same time.

The `git appraise pull` command does not do any sort of locking to prevent two invocations happening at the same time (although, now that I think of it, it probably should).

It would be reasonable to push that change off to a subsequent PR, but I do suspect we will need to eventually do it.

Reply:
This change will be incorporated in a future PR.